### PR TITLE
POST messages now sends text message

### PR DESCRIPTION
**Pull request by clark-teeple**

### CODE REVIEW

File: src/api/handlers/messages/postMessages.js [hunk 1]
```diff
@@ -37,16 +50,26 @@ export default async (req, res) => {
...
+      error,
+      message: "Could not send text reminder"
+    });
+    return;
   }
 
-  // TODO: Send sms message
-  // Not sure if we want to create a transation and only commit once we get confirmation from twilio
-  // Or write the message to the DB and include a "successful" field, updating after twilio response
+  try {
```
Comment (iamstevedavis) [inline src/api/handlers/messages/postMessages.js:62]: I think this should be something like
```
try {
  await db("messages").insert(message);
} catch (error) {
  res.status(500).send({
    error,
    message: "Could not save message to database"
  });
  return;
}

res.status(201).send(message);
```
I think you might be mixing sync/async code here. You can verify the message actually inserted if you want too with
```
let insertedId = null;
try {
insertedId = db("messages").insert(message, ["id"])
...
```

Comment (clark-teeple) [reply src/api/handlers/messages/postMessages.js:62]: Would it be the same thing on line 24 then?

Comment (iamstevedavis) [reply src/api/handlers/messages/postMessages.js:62]: Oh, yeah, I missed that before. Good catch!

Comment (iamstevedavis) [reply src/api/handlers/messages/postMessages.js:62]: You can also use http://knexjs.org/#Builder-returning as a more concise way of defining a return

Comment (clark-teeple) [reply src/api/handlers/messages/postMessages.js:62]: lol I added that earlier... documentation says mysql is supported, mysql says `.returning() is not supported by mysql and will not have any effect.`

Comment (iamstevedavis) [reply src/api/handlers/messages/postMessages.js:61]: Well shut my mouth.

**Pull request merged**